src/xcode/ENA/ENA/Source/Scenes/ENSetting/Cells/DaysSinceInstallTableViewCell.swift

```diff
@@ -48,7 +48,7 @@ class DaysSinceInstallTableViewCell: UITableViewCell {
 			self.heightDidChange?()
 		}
 		
-		if viewModel.daysSinceInstall < 14 {
+		if viewModel.daysSinceInstall < 11 {
```

Comment:
Should it be 10?

Reply:
okay current discussion: threshold value for exposure logging = 14 days, exposure detection = 10 days. the bug description seems to be wrong. when the bug will be consistent, i will adapt the code as well again

Reply:
Can you please update the PR description?

Reply:
done